cpp/oneapi/dal/algo/triangle_counting/test/triangle_counting_test.cpp

```diff
@@ -323,12 +323,12 @@ class triangle_counting_test {
         std::int64_t global_triangle_count = graph_data.get_global_triangle_count();
 
         std::allocator<char> alloc;
```

Comment:
There is no way to avoid creating allocator object because triangle_counting descriptor's constructor needs it: https://github.com/oneapi-src/oneDAL/blob/master/cpp/oneapi/dal/algo/triangle_counting/common.hpp#L110